cves/CVE-2004-0493.yml

```diff
@@ -94,10 +97,10 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
-  date:
-  automated:
-  google:
+  answer: I searched the references on cve.mitre.org, but was unable to find a method of discovery
```

Comment:
Discovered by Georgi Guninski (see https://github.com/apache/httpd/commit/3d5263c6dcdf517a9eb5a0ab916e1ad49d354355), and was reported on 13th June 2004 (see http://httpd.apache.org/security/vulnerabilities_20.html).